packages/ia-components/package.json

```
@@ -23,7 +23,7 @@
     "@babel/plugin-transform-runtime": "^7.5.5",
     "@babel/preset-env": "^7.13.10",
     "@babel/runtime": "^7.5.5",
-    "@internetarchive/bookreader": "^4.37.0",
+    "@internetarchive/bookreader": "^5.0.0-0",
```

We want this one pinned; we'll likely be making breaking changes to this and don't want to it to autoupdate. Also, I messed up, and my experimental `5.0.0-beta-18ceefd` matches over `^5.0.0-0` :( Need to find a fix for that.

```suggestion
    "@internetarchive/bookreader": "5.0.0-1",
```

---

hi drini thanks for the suggestion. this must stay in lock-step with latest bookreader, so it should be ok to use caret ^_^